packages/ember-template-compiler/lib/plugins/deprecate-view-and-controller-paths.js

```diff
@@ -55,7 +55,7 @@ function deprecatePaths(moduleName, node, paths) {
 }
 
 function deprecatePath(moduleName, node, path) {
-  Ember.deprecate(`Using \`{{${path && path.type === 'PathExpression' && path.parts[0]}}}\` or any path based on it ${calculateLocationDisplay(moduleName, node.loc)}has been deprecated.`, !(path && path.type === 'PathExpression' && (path.parts[0] === 'view' || path.parts[0] === 'controller')), { url: 'http://emberjs.com/deprecations/v1.x#toc_view-and-controller-template-keywords', id: 'view-controller-keyword' });
+  Ember.deprecate(`Using \`{{${path && path.type === 'PathExpression' && path.parts[0]}}}\` or any path based on it ${calculateLocationDisplay(moduleName, node.loc)}has been deprecated.`, !(path && path.type === 'PathExpression' && (path.parts[0] === 'view' || path.parts[0] === 'controller')), { url: 'http://emberjs.com/deprecations/v1.x#toc_view-and-controller-template-keywords', id: (path.parts && path.parts[0] === 'view' ? 'view.keyword.view' : 'view.keyword.controller') });
```

Comment:
this ensures that `{{view}}` keyword gets a deprecation id of `view.keyword.view` and `{{controller}}` keyword gets a different deprecation id of `view.keyword.controller`.